node/express/containerCosmosDBWithTests/Tests/package.json

```diff
@@ -7,7 +7,7 @@
   },
   "dependencies": {
     "applicationinsights": "^1.0.0",
-    "gulp": "^3.9.1",
+    "gulp": "^4.0.2",
```

Comment:
This is breaking change, did you test if app is deployed?

Reply:
fixed that

Reply:
verified it using this DevOps project: https://portal.azure.com/#@microsoft.onmicrosoft.com/resource/subscriptions/c00d16c7-6c1f-4c03-9be1-6934a4c49682/resourcegroups/VstsRG-hiyada-7247/providers/Microsoft.DevOps/pipelines/nodeexpresswithcosmos